tests/run_integration_tests.sh

```diff
@@ -53,7 +50,13 @@ function clone_llama_stack() {
 }
 
 function run_integration_tests() {
-    echo "Running integration tests..."
+    # Check if model is provided
+    if [ -z "$1" ]; then
+        echo "Error: No model provided"
+        exit 1
+    fi
+    local model="$1"
```

Comment:
this code is repeated pretty frequently, you should probably just make it a function